lib/cli.js

```diff
@@ -47,44 +49,58 @@ module.exports = function parseCLI(argv = process.argv.slice(2), help = HELP) {
 				"mode to avoid littering your file system with obsolete bundles");
 	}
 
-	let { source, target } = argv;
-	// TODO: allow users to overwrite this in a custom faucet.config.js
-	let baseURI = "/";
-	let config = {
-		aiur: [{
-			source,
-			target,
-			baseURI
-		}],
+	let assetConfig = path.resolve(referenceDir, "faucet.config.js"); // TODO: customizable?
+	try {
+		assetConfig = require(assetConfig);
+	} catch(err) {
+		if(err.code !== "MODULE_NOT_FOUND") {
+			throw err;
+		}
+		assetConfig = {};
+	}
 
-		// TODO: make this configurable
-		sass: [{
-			source: "aiur/lib/style.scss",
-			target: `${target}/style.css`
-		}],
+	let { source, target, watch, fingerprint, sourcemaps, compact } = argv;
+	return {
+		referenceDir,
+		config: generateConfig(source, target, referenceDir, assetConfig),
+		options: { watch, fingerprint, sourcemaps, compact }
+	};
+};
 
-		// TODO: allow users to overwrite this in a custom faucet.config.js
-		watchDirs: [source, "./components"],
+function generateConfig(source, target, referenceDir, assetsConfig) {
+	let componentsDir = "./components"; // TODO: customizable
+	let baseURI = "/"; // TODO: customizable
 
-		manifest: {
-			baseURI,
-			webRoot: target
-		},
+	let { watchDirs, sass, plugins } = assetsConfig;
 
-		plugins: {
-			aiur: {
-				plugin: aiur,
-				bucket: "markup"
-			}
-		}
-	};
+	// merge `watchDirs`
+	let defaults = [source, componentsDir];
+	watchDirs = watchDirs ? defaults.concat(watchDirs) : defaults;
+
+	// merge `sass`
+	defaults = [{
+		source: "aiur/lib/style.scss", // TODO: customizable?
+		target: `${target}/style.css`
```

Comment:
`s/style.css/aiur.css/`?

Reply:
Yep, please rename this to `aiur.css`

Reply:
9b408ad5a1a1611cbea43b6c50e6101bf4c02711